samples/fuel_gauge/Kconfig

```diff
@@ -0,0 +1,194 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+mainmenu "Fuel Gauge example"
+	choice
+		prompt "VBUSIN current limit"
+		default CURRENT_LIMIT_500
+
+		config CURRENT_LIMIT_100
+			bool "100 mA"
+		config CURRENT_LIMIT_500
+			bool "500 mA"
+		config CURRENT_LIMIT_600
+			bool "600 mA"
+		config CURRENT_LIMIT_700
+			bool "700 mA"
+		config CURRENT_LIMIT_800
+			bool "800 mA"
+		config CURRENT_LIMIT_900
+			bool "900 mA"
+		config CURRENT_LIMIT_1000
+			bool "1000 mA"
+		config CURRENT_LIMIT_1100
+			bool "1100 mA"
+		config CURRENT_LIMIT_1200
+			bool "1200 mA"
+		config CURRENT_LIMIT_1300
+			bool "1300 mA"
+		config CURRENT_LIMIT_1400
+			bool "1400 mA"
+		config CURRENT_LIMIT_1500
+			bool "1500 mA"
+	endchoice
+
+	config CURRENT_LIMIT
+		int
+		default 100 if CURRENT_LIMIT_100
+		default 500 if CURRENT_LIMIT_500
+		default 600 if CURRENT_LIMIT_600
+		default 700 if CURRENT_LIMIT_700
+		default 800 if CURRENT_LIMIT_800
+		default 900 if CURRENT_LIMIT_900
+		default 1000 if CURRENT_LIMIT_1000
+		default 1100 if CURRENT_LIMIT_1100
+		default 1200 if CURRENT_LIMIT_1200
+		default 1300 if CURRENT_LIMIT_1300
+		default 1400 if CURRENT_LIMIT_1400
+		default 1500 if CURRENT_LIMIT_1500
+
+	menu "Battery configuration"
+		config CHARGING_CURRENT
+			int "Battery charging current (in milliamperes)"
+			range 32 800
+			default 150
+
+		config DISCHARGING_CURRENT
+			int "Battery discharge current limit (in milliamperes)"
+			help
+				Available range is 270 mA to 1340 mA in 3.23 mA steps.
+				The value specified will be rounded down to the closest implemented value.
+			range 270 1340
```

Comment:
We have a note about 3.23 mA step in `DISCHARGING_CURRENT`. It may be worth to have a similar note about 2 mA steps under `CHARGING_CURRENT`